controller/iteration_blackbox_test.go

```diff
@@ -78,8 +78,7 @@ func (rest *TestIterationREST) TestCreateChildIteration() {
 
 	rest.T().Run("success - create child iteration", func(t *testing.T) {
 		fxt := tf.NewTestFixture(t, rest.DB,
-			tf.Identities(1),
-			tf.Areas(1),
+			tf.CreateWorkItemEnvironment(),
 			tf.Iterations(2,
 				tf.SetIterationNames("root iteration", "child iteration"),
 				tf.PlaceIterationUnderRootIteration()))
```

Comment:
No need to this line. It is already done when you set up your environment.

Reply:
yes done in https://github.com/fabric8-services/fabric8-wit/pull/1737/commits/b6f75a5788b046d426341f0eb234e8b6f99e4b3f